dicoogle/src/main/java/pt/ua/dicoogle/server/queryretrieve/CMoveServiceSCP.java

```diff
@@ -190,8 +179,7 @@ protected DimseRSP doCMove(Association as, int pcid, DicomObject cmd,
 
 
         if (search == null) {
-            //DebugManager.getInstance().debug(">> Search is null, so"
-            //        + " somethig is wrong ");
+            logger.error(">> Search is empty/null, somethig is wrong with query terms");
         } else {
```

Comment:
`search` can never be null, it was assigned via `new`. The condition can just be removed.